third_party/cached_httpfs/include/httpfs.hpp

```diff
@@ -194,6 +201,7 @@ class HTTPFileSystem : public FileSystem {
 	// Global cache
 	mutex global_cache_lock;
 	duckdb::unique_ptr<HTTPMetadataCache> global_metadata_cache;
+	duckdb::unique_ptr<HTTPFileCache> glob_file_cache;
```

Comment:
nit: `global_file_cache`

"glob" is a separate thing and could be confusing here